js/react-runner/Dockerfile

```diff
@@ -24,11 +24,20 @@ FROM app0
 
 # TODO: make this less flimsy
 COPY --from=stdlib /stdlib/*.mjs /stdlib/package.json \
-     /app/node_modules/@reach-sh/stdlib/
-RUN sed -i.bak '/ganache-core/d' '/app/node_modules/@reach-sh/stdlib/package.json' \
-  && rm '/app/node_modules/@reach-sh/stdlib/package.json.bak'
-RUN sed -i.bak "s/import('ganache-core')/null/" '/app/node_modules/@reach-sh/stdlib/ETH.mjs' \
-  && rm '/app/node_modules/@reach-sh/stdlib/ETH.mjs.bak'
+     /stdlib/
+RUN sed -i.bak '/ganache-core/d' '/stdlib/package.json' \
+  && rm '/stdlib/package.json.bak'
+RUN sed -i.bak "s/import('ganache-core')/null/" '/stdlib/ETH.mjs' \
+  && rm '/stdlib/ETH.mjs.bak'
+RUN npm install /stdlib
```

Comment:
DONOTMERGE until this stuff has been tested. I think `react-runner` has fallen behind from its sibling `runner`